src/Trail/Trail.fs

```diff
@@ -53,6 +56,21 @@ module Dom =
     
     let text textContent = Text textContent
     let textf format content = Text(Printf.sprintf format content)
+
+module Attr =
+    open Dom
+    
+    let namedUIEvent name (handler:string) =
+        BlazorEventAttribute(name, BindMethods.GetEventHandlerValue(handler))
+    
+    let uiEventHandler name handler =
+        BlazorEventHandlerAttribute(name, BindMethods.GetEventHandlerValue(Action handler))
+
+    let typedUIEventHandler<'T when 'T :> UIEventArgs> name handler =
+        BlazorEventHandlerAttribute(name, BindMethods.GetEventHandlerValue(Action<'T> handler))
+    
+    let onclick handler =
+        typedUIEventHandler<UIMouseEventArgs> "onclick" handler
```

Comment:
There are more examples in the [blog post](https://blogs.msdn.microsoft.com/webdev/2018/04/17/blazor-0-2-0-release-now-available/) that I should probably add.